gensim/models/fasttext.py

```diff
@@ -190,7 +190,7 @@
     True
 
 If you do not intend to continue training the model, consider using the
-:func:`gensim.models.FastText.load_facebook_vectors` function instead.
+:func:`gensim.models.fastText.load_facebook_vectors` function instead.
```

Comment:
Should it be

```suggestion
:func:`gensim.models.fasttext.load_facebook_vectors` function instead.
```

instead?

Reply:
RIght